src/tests/stdlib/randint.ab

```diff
@@ -0,0 +1,4 @@
+import * from "std"
+main {
+    echo randint(1,1)
```

Comment:
this does not check if any weird range would output correct info, like if min would be greater than max

Reply:
I made 1 random out of 1 because how to guess the result to pass the tests.